sdks/npm/src/cf/on-ua-applicator.ts

```diff
@@ -8,8 +8,12 @@ export default class UATransformApplicator  implements HTMLRewriterElementConten
   }
 
   static serializeOperation(operation: Experimentation.Operations, args: any[]) {
-    if (operation === Experimentation.Operations.CustomJs) {
-      return [operation, `$=>{${args[0]}}`];
+    switch (operation) {
+      case Experimentation.Operations.CustomJs:
+        return [operation, `$=>{${args[0]}}`];
+      case Experimentation.Operations.Redirect:
+        // In case of a redirect, we use a CustomJs operation to rewrite location.
+        return [Experimentation.Operations.CustomJs, `$=>{window.location.href="${JSON.stringify(args[0])}"}`];
```

Comment:
We'll need to validate our `args` eventually, for all of our operations, for both PRE and ON UA. I don't think that's necessary for this PR.

Reply:
Added this ticket to resolve this: https://github.com/WICG/ab-worker-prototype/issues/20